marker_api/src/ast/ty.rs

```diff
@@ -97,26 +97,58 @@ pub enum TyKind<'ast> {
     // ================================
     /// The `bool` type
     Bool(&'ast BoolTy<'ast>),
-    /// A numeric type like `u32`, `i32`, `f64`
+    /// A numeric type like [`u32`], [`i32`], [`f64`]
     Num(&'ast NumTy<'ast>),
-    /// A textual type like `char` or `str`
+    /// A textual type like [`char`] or [`str`]
     Text(&'ast TextTy<'ast>),
-    /// The never type `!`
+    /// The never type [`!`][prim@never]
     Never(&'ast NeverTy<'ast>),
     // ================================
     // Sequence types
     // ================================
-    /// A tuple type like `()`, `(T, U)`
+    /// A tuple type like [`()`][prim@tuple], [`(T, U)`][prim@tuple]
     Tuple(&'ast TupleTy<'ast>),
-    /// An array with a known size like: `[T; n]`
+    /// An array with a known size like: [`[T; N]`][prim@array]
     Array(&'ast ArrayTy<'ast>),
-    /// A variable length slice like `[T]`
+    /// A variable length slice like [`[T]`][prim@slice]
```

Comment:
Also, these doc comments. The `prim@` link is actually interesting. Probably one of these things that very few projects run into. And I'm happy to be part of one of these projects :joy:

Reply:
Haha, yeah, glad to see that my "random looking into stdlib source" pays off. A bit sad that there isn't a shortcut like this for keywords, only for primitive types :(